composer.json

```
@@ -18,16 +18,16 @@
         "source": "https://github.com/cakephp/queue"
     },
     "require": {
-        "php": ">=7.2.0",
-        "cakephp/cakephp": "^4.1",
+        "php": ">=7.4.0",
+        "cakephp/cakephp": ">=4.1 ",
         "enqueue/simple-client": "^0.10",
-        "psr/log": "^1.1"
+        "psr/log": "^1.0 || ^2.0"
```

```suggestion
        "psr/log": "^1.1 || ^2.0"
```
Except for this the change for all other packages are unnecessary.

---

Agree with @ADmad. There is no reason to pin these higher, as we also test the low end of our compatibility ranges. Having wider compatibility ranges means that more users can benefit from these changes as we get exposure to more versions.